ui/components/ui/box/box.stories.js

```diff
@@ -545,84 +556,85 @@ export const BorderRadius = () => {
-          borderColor={COLORS.BORDER_DEFAULT}
+          backgroundColor={BackgroundColor.backgroundAlternative}
+          borderColor={BorderColor.borderDefault}
           borderWidth={2}
-          borderRadius={BORDER_RADIUS.SM}
+          borderRadius={BorderRadius.SM}
         >
-          BORDER_RADIUS.SM 4px
+          BorderRadius.SM 4px
         </Box>
         <Box
           padding={3}
-          backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
-          borderColor={COLORS.BORDER_DEFAULT}
+          backgroundColor={BackgroundColor.backgroundAlternative}
+          borderColor={BorderColor.borderDefault}
           borderWidth={2}
-          borderRadius={BORDER_RADIUS.MD}
+          borderRadius={BorderRadius.MD}
         >
-          BORDER_RADIUS.MD 6px
+          BorderRadius.MD 6px
         </Box>
         <Box
           padding={3}
-          backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
-          borderColor={COLORS.BORDER_DEFAULT}
+          backgroundColor={BackgroundColor.backgroundAlternative}
+          borderColor={BorderColor.borderDefault}
           borderWidth={2}
-          borderRadius={BORDER_RADIUS.LG}
+          borderRadius={BorderRadius.LG}
         >
-          BORDER_RADIUS.LG 8px
+          BorderRadius.LG 8px
         </Box>
         <Box
           padding={3}
-          backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
-          borderColor={COLORS.BORDER_DEFAULT}
+          backgroundColor={BackgroundColor.backgroundAlternative}
+          borderColor={BorderColor.borderDefault}
           borderWidth={2}
-          borderRadius={BORDER_RADIUS.XL}
+          borderRadius={BorderRadius.XL}
         >
-          BORDER_RADIUS.XL 12px
+          BorderRadius.XL 12px
         </Box>
         <Box
           padding={3}
-          backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
-          borderColor={COLORS.BORDER_DEFAULT}
+          backgroundColor={BackgroundColor.backgroundAlternative}
+          borderColor={BorderColor.borderDefault}
           borderWidth={2}
-          borderRadius={BORDER_RADIUS.PILL}
+          borderRadius={BorderRadius.pill}
         >
-          BORDER_RADIUS.PILL 9999px
+          BorderRadius.pill 9999px
         </Box>
       </Box>
       <Box
         padding={3}
-        backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
-        borderColor={COLORS.BORDER_DEFAULT}
+        backgroundColor={BackgroundColor.backgroundAlternative}
+        borderColor={BorderColor.borderDefault}
         borderWidth={2}
-        borderRadius={BORDER_RADIUS.FULL}
+        borderRadius={BorderRadius.full}
         margin={4}
         display={DISPLAY.FLEX}
-        alignItems={ALIGN_ITEMS.CENTER}
+        alignItems={AlignItems.center}
         style={{ height: '250px', width: '250px' }}
       >
-        BORDER_RADIUS.FULL 50%
+        BorderRadius.full 50%
       </Box>
     </>
   );
 };
+BorderRadiusStories.storyName = 'BorderRadius';
```

Comment:
Can you revert this change on this PR and make it separately?``

Reply:
Actually It is needed to avoid name conflict ~
https://github.com/MetaMask/metamask-extension/pull/17518#discussion_r1093507043